backend/src/main/java/com/example/backend/controller/UserController.java

```diff
@@ -0,0 +1,24 @@
+package com.example.backend.controller;
+
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+
+    @GetMapping("/me")
+    public String getMe() {
+        return SecurityContextHolder.getContext().getAuthentication().getName();
+    }
+    @PostMapping("/login")
+    public String login() {
+        return SecurityContextHolder.getContext().getAuthentication().getName();
+    }
```

Comment:
we asked ourselves if this request is working, because its the same like the getMe

Reply:
The first one is a GET-Request and this one a POST-Request